client/lib/analytics/fullstory.js

```diff
@@ -56,6 +65,11 @@ function maybeAddFullStoryScript() {
 		return;
 	}
 
+	// Only record session for US-based users.
+	if ( 'US' !== cookie.parse( document.cookie ).country_code ) {
```

Comment:
Any reason to limit to the US only, or would English locale work better?

Reply:
I just went this way to further limit any potential GDPR risks, but can check for EN instead if it isn't an issue.

Reply:
I know we can manage regions from the FullStory dashboard so we're probably safe without this check. That said, it was mentioned at one point to have the additional check in code for extra security.

Let's get some clarification on how we're planning to manage this and if it's necessary at all so we can better determine what to check for.

(cc: @davemart-in)